include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,359 @@
+     * \f]
+     *
+     *Coefficients a_mix, b_mix and (a \alpha)_{mix} are caclulated as
+     *
+     *\f[
+     *a_{mix} = \sum_i \sum_j X_i X_j a_{i, j} = \sum_i \sum_j X_i X_j sqrt{a_i a_j}
+     *\f]
+     *
+     *\f[
+     *   b_{mix} = \sum_i X_i b_i
+     *\f]
+     *
+     *\f[
+     *   {a \alpha}_{mix} = \sum_i \sum_j X_i X_j {a \alpha}_{i, j} = \sum_i \sum_j X_i X_j sqrt{a_i a_j} sqrt{\alpha_i \alpha_j}
+     *\f]
+     *
+     *
+     */
+    
+    virtual double pressure() const;
+
+    // @}
+
+public:
+
+    //! Returns the standard concentration \f$ C^0_k \f$, which is used to
+    //! normalize the generalized concentration.
+    /*!
+     * This is defined as the concentration by which the generalized
+     * concentration is normalized to produce the activity. In many cases, this
+     * quantity will be the same for all species in a phase. 
+     * The ideal gas mixture is considered as the standard or reference state here. 
+     * Since the activity for an ideal gas mixture is simply the mole fraction, for an ideal gas
+     * \f$ C^0_k = P/\hat R T \f$.
+     *
+     * @param k Optional parameter indicating the species. The default is to
+     *          assume this refers to species 0.
+     * @return
+     *   Returns the standard Concentration in units of m3 kmol-1.
+     */
+    virtual double standardConcentration(size_t k=0) const;
+
+    //! Get the array of non-dimensional activity coefficients at the current
+    //! solution temperature, pressure, and solution concentration.
+    /*!
+     * For all objects with the Mixture Fugacity approximation, we define the
+     * standard state as an ideal gas at the current temperature and pressure of
+     * the solution. The activities are based on this standard state.
+     *
+     * @param ac Output vector of activity coefficients. Length: m_kk.
+     */
+    virtual void getActivityCoefficients(double* ac) const;
+
+    /// @name  Partial Molar Properties of the Solution
+    //@{
+
+    //! Get the array of non-dimensional species chemical potentials.
+    //! These are partial molar Gibbs free energies.
+    /*!
+     * \f$ \mu_k / \hat R T \f$.
+     * Units: unitless
+     *
+     * We close the loop on this function here calling getChemPotentials() and
+     * then dividing by RT. No need for child classes to handle.
+     *
+     * @param mu    Output vector of non-dimensional species chemical potentials
+     *              Length: m_kk.
+     */
+    
+    virtual void getChemPotentials(double* mu) const;
+    virtual void getPartialMolarEnthalpies(double* hbar) const;
+    virtual void getPartialMolarEntropies(double* sbar) const;
+    virtual void getPartialMolarIntEnergies(double* ubar) const;
+    virtual void getPartialMolarCp(double* cpbar) const;
+    virtual void getPartialMolarVolumes(double* vbar) const;
+
+    //! Calculate the temperature dependent interaction parameter alpha needed for P-R EoS
+    /*

```

Comment:
This docstring isn't formatted so that it shows up in the Doxygen docs, I think you're missing a bang here (`!`) and then all the following lines need to be indented by one more space so the `*` lines up. I'm not sure which of those changes fixed the problem, but one of them did :smiley:

Reply:
Corrected.